app/src/main/java/com/pajato/android/gamechat/database/ExperienceManager.java

```diff
@@ -183,11 +183,11 @@ public Experience getExperience(@NonNull final String groupKey, @NonNull final S
             case 0:
             case 1:             // Get the experiences from the rooms in the joined group that have
                                 // experiences and the me room if it has any experiences.
-                String roomKey = expGroupMap.keySet().iterator().next();
+                String groupKey = expGroupMap.keySet().iterator().next(); // this is actually a group key ?!!!??????????????
```

Comment:
Oops. Thought I removed this, but it looks like it didn't go through. I'll grab it next commit.